examples/layer-browser/src/examples/core-layers.js

```diff
@@ -67,6 +67,40 @@ const IconLayerExample = {
   }
 };
 
+const IconLayerDynamicIconsExample = {
+  layer: IconLayer,
+  getData: () => dataSamples.points,
+  props: {
+    id: 'icon-layer-experimental',
+    sizeScale: 24,
+    getPosition: d => d.COORDINATES,
+    getColor: d => [64, 64, 72],
+    getIcon: d => {
+      if (d.PLACEMENT === 'SW') {
+        return {
+          url: 'data/icon-marker.png',
+          width: 128,
+          height: 128,
+          anchorY: 128,
+          mask: true
+        };
+      }
+      return {
+        url: 'data/icon-warning.png',
+        width: 128,
+        height: 128,
+        anchorY: 128,
+        mask: false
+      };
+    },
+    getSize: d => {
+      return d.RACKS > 2 ? 2 : 1;
```

Comment:
what should this method return? Scale of the icon?

Reply:
https://github.com/uber/deck.gl/blob/master/docs/layers/icon-layer.md#getsize-functionnumber-optional-